sway/input/keyboard.c

```diff
@@ -5,6 +5,7 @@
 #include <wlr/backend/session.h>
 #include <wlr/interfaces/wlr_keyboard.h>
 #include <wlr/types/wlr_idle.h>
+#include <wlr/types/wlr_keyboard.h>
```

Comment:
Note for reviewer: already implicitly included through `wlr_keyboard_group.h` below, but probably best to make it explicit.